atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueServiceTest.java

```diff
@@ -736,6 +738,80 @@ public void testDeleteMultipleVersions() {
     }
 
     @Test
+    public void testDeleteRangeReverse() {
+        if (reverseRangesSupported()) {
+            putTestDataForMultipleRows(); // set up rows row0,row1,row2
+            RangeRequest range = RangeRequest.builder(true)
+                    .startRowInclusive("row1b".getBytes())
+                    .endRowExclusive(row0)
+                    .build();
+            // should delete only row1
+            keyValueService.deleteRange(TEST_TABLE, range);
+
+            List<byte[]> keys = Lists.newArrayList();
+            keyValueService.getRange(TEST_TABLE, RangeRequest.all(), Long.MAX_VALUE)
```

Comment:
Should use `MAX_TS` (first of 5 times in this class, but you can get it down to 1 or 2 via the above refactor)